### PR TITLE
Add accounts to /etc/hosts when running configure_hosts.yml

### PR DESCRIPTION
**Pull request by karenc**

### CODE REVIEW

File: configure_hosts.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     - pdf_gen
   tasks:
     - name: get dns from local /etc/hosts
-      local_action: "command grep -E '^${% for group in groups.all %}|{{ '\\<%s\\>'|format(group) }}{% endfor %}' /etc/hosts"
+      local_action: "command grep -E '\\<{{ accounts_domain }}\\>{% for group in groups.all %}|{{ '\\<%s\\>'|format(group) }}{% endfor %}' /etc/hosts"
```
Comment (pumazi) [inline configure_hosts.yml]: Can you explain what this is doing?

Comment (karenc) [inline configure_hosts.yml]: It used to be:

```
grep -E "^$|\<local.cnx.org\>|\<zope.local.cnx.org\>|\<127.0.0.1\>" /etc/hosts
```

It is now:

```
grep -E "\<accounts.local.openstax.org\>|\<local.cnx.org\>|\<zope.local.cnx.org\>|\<127.0.0.1\>" /etc/hosts
```

getting the `/etc/hosts` entries for `local.cnx.org` (like in python `\blocal.cnx.org\b`) and `zope.local.cnx.org`.  Then I realized it's not looking for `accounts.local.openstax.org` which I needed in the VM.  Does that answer your question?

Comment (pumazi) [inline configure_hosts.yml]: Yes, thanks.

 I'm guessing grep was likely easier to debug than the `lineinfile` module. So, good choice.

Comment (karenc) [inline configure_hosts.yml]: This is actually the only way I know how to get the `/etc/hosts` entries from localhost :sweat_smile:

Comment (pumazi) [inline configure_hosts.yml]: O... actually, I misunderstood anyway. I get it now. 👍

**Pull request merged**